src/Docker/OutputFilter/OutputFilter.php

```diff
@@ -38,10 +45,23 @@ public function collectValues(array $data)
      * @inheritdoc
      */
     public function filter($text)
+    {
+        return $this->filterSecrets($this->filterGarbage((string) $text));
```

Comment:
Does this maybe leak partial secrets?

Reply:
That's a good point probably, fixed in https://github.com/keboola/docker-bundle/pull/641/commits/0d17da21c6acab6aa00c5394f231bb224a605f49

Reply:
the only thing about filtering first, wasn't that the whole issue to begin with?

Reply:
ah, no, it was after that, during the encoding conversion.  sorry